app/services/invoices/subscription_service.rb

```diff
@@ -139,7 +139,10 @@ def flag_lifetime_usage_for_refresh
     end
 
     def tax_error?(fee_result)
-      !fee_result.success? && fee_result&.error&.code == 'tax_error'
+      return false if fee_result.success?
```

Comment:
There's 7 implementors of `tax_error?` and looks like a total of 3 different implementations. Maybe a simple module that's included in BaseService? Or a class method somewhere? 🤔 Just so we can align all implementations maybe? what do you think?

Reply:
I agree. This will be tacled in a dedicated pull request as it might require some deeper changes